demo/frontend/package.json

```diff
@@ -24,24 +18,27 @@
     "styled-components": "6.1.1"
   },
   "devDependencies": {
+    "@biomejs/biome": "^2.1.1",
     "@types/node": "^20.9.0",
     "@types/react": "^18.2.37",
     "@types/react-dom": "^18.2.15",
-    "@vitejs/plugin-react": "^4.1.1",
-    "eslint": "8.53.0",
-    "eslint-config-airbnb": "19.0.4",
-    "eslint-config-airbnb-base": "15.0.0",
-    "eslint-config-react-app": "7.0.1",
-    "typescript": "^5.2.2",
-    "vite": "^4.5.14",
-    "vite-plugin-mkcert": "^1.16.0",
-    "vite-tsconfig-paths": "^4.2.1"
+    "@vitejs/plugin-react": "^4.6.0",
+    "typescript": "^5.8.3",
+    "vite": "^7.0.0",
+    "vite-plugin-mkcert": "^1.17.8",
+    "vite-tsconfig-paths": "^5.0.0"
```

Comment:
I had to bump vite a bit because are some incompatabilities with modern nodejs versions in terms of type-checking.

I can break this out to another PR, but this is fairly light in terms of lines changed in the ts/vite config files.